### PR TITLE
Update Japanese version commands

### PR DESCRIPTION
**Pull request by toshi0607**
Original version commans were updated so Japanese version should also be updated.

### CODE REVIEW

File: translations/ja/lab5.md [hunk 1]
```diff
@@ -67,10 +65,10 @@ functions:
       write_debug: true
 ```
 
-* functionをデプロイします
+* functionをビルド、プッシュ、デプロイします
 
 ```
-$ faas-cli deploy -f ./issue-bot.yml
```
Comment (alexellis) [inline translations/ja/lab5.md]: This one probably shouldn't change

Comment (toshi0607) [reply translations/ja/lab5.md]: The line is deleted in this commit.
https://github.com/openfaas/workshop/commit/f27ff19c87b0933c51799586d83d5e75cbd78eb2#diff-9852b83ba9abfceda99773422ef371ac

And I think it will be deployed L72.
```
$ faas-cli up -f ./issue-bot.yml
```

Comment (kenfdev) [reply translations/ja/lab5.md]: It looks like the original version has changed, too.
https://github.com/openfaas/workshop/commit/f27ff19c87b0933c51799586d83d5e75cbd78eb2#diff-9852b83ba9abfceda99773422ef371acR72

But yes, this only needs `deploy` because the environment variable is the only thing that changed. Perhaps we should revert it back to `faas-cli deploy -f ./issue-bot.yml`.

**Comment by kenfdev** [comment]
LGTM

File: lab5.md [hunk 2]
```diff
@@ -66,10 +66,10 @@ functions:
       write_debug: true
 ```
 
-* Build, push and deploy the function with
+* Deploy the function
 
 ```
-$ faas-cli up -f ./issue-bot.yml
```
Comment (alexellis) [inline lab5.md]: Why did you remove `faas-cli up`?

Comment (kenfdev) [reply lab5.md]: @alexellis This is getting confusing but started from this discussion:
https://github.com/openfaas/workshop/pull/78#discussion_r224955735

I requested to revert the change:
https://github.com/openfaas/workshop/pull/78#discussion_r224955904

But actually, this instruction is creating the function right before this place so it actually needs the `faas-cli up`. If that's true, https://github.com/openfaas/workshop/pull/78/commits/d6e5d164962f8f625b3c9d933fdaa2118ea5a371 needs to be reverted. Alex, could you confirm?

@toshi0607 sorry about my misleading comment but could you revert the commit after Alex confirms?

**Comment by alexellis** [comment]
@kenfdev please can you let me know if you want this merged?

File: lab5.md (hunk 2 above)
Comment (kenfdev) [reply lab5.md]: @toshi0607 Sorry about the totally late response. Could you please revert this to `$ faas-cli up -f ./issue-bot.yml` ? So that it matches the original version.
Sorry about the back and forth 😢

File: translations/ja/lab5.md [hunk 3]
```diff
@@ -67,10 +65,10 @@ functions:
       write_debug: true
 ```
 
-* functionをデプロイします
+* functionをbuild、push、deployしましょう
```
Comment (toshi0607) [inline translations/ja/lab5.md]: Original version says

```
Build, push and deploy the function with
```
https://github.com/openfaas/workshop/blob/master/lab5.md

**Comment by kenfdev** [comment]
Thanks @toshi0607 !

@alexellis This PR is GTG. The Japanese version has to catch up with the original version, but this PR is good individually.

**Comment by alexellis** [comment]
I don't want to merge this pull request because it has far too many commits that include applying changes and then reverting them. Thank you for helping us maintain the Japanese translation, but please could you squash these commits into a single commit and then notify me?

**Comment by alexellis** [comment]
Thank you both :-)
